app/models/concerns/reviews/approval.rb

```diff
@@ -48,7 +48,7 @@ def review_errors
         review_errors << I18n.t('review.errors.without_survey')
       end
 
-      if file_model.blank? && SHOW_REVIEW_EXTRA_ATTRIBUTES
+      if file_models.any? && SHOW_REVIEW_EXTRA_ATTRIBUTES
```

Comment:
```suggestion
      if file_models.empty? && SHOW_REVIEW_EXTRA_ATTRIBUTES
```

Sino queda la pregunta al revés.